### ISSUE TITLE
this _repository_

### ISSUE THREAD

**Issue by demhydraz** [Piorjade/cLinux#11]

**Comment by daelvn** [comment]
I was thinking that myself, maybe Pior can look onto it?
It is true that the `Folder/File` doesn't look very nice on it

**Comment by GReaperEx** [comment]
What's up with the trolling issues and the changing? Do you know each other?

**Comment by demhydraz** [comment]
this is shit

**Comment by daelvn** [comment]
@GReaperEx Yes, he made the original issues, I changed it to actual bugs that I pointed out and need to be fixed. So, this actually would have to be taken as an issue
